documentation/modules/configuring/proc-altering-connector-offsets.adoc

```diff
@@ -0,0 +1,127 @@
+// Module included in the following assemblies:
+//
+// assembly-config.adoc
+
+[id='proc-altering-connector-offsets-{context}']
+= Altering connector offsets
+
+[role="_abstract"]
+To alter connector offsets using `KafkaConnector` resources, add the `alterOffsets` configuration.
+This can be useful, for example, to skip a _poison_ record or replay a record.
+The offset changes are specified in a config map. 
+You can reuse the same config map used to xref:proc-listing-connector-offsets-{context}[list offsets].
+The offset alteration is applied by annotating the `KafkaConnector` resource.
+
+In this procedure, we alter the offset position for a source connector named `my-source-connector`.
+
+NOTE: You can also alter connector offsets using the `PATCH /connectors/{connector}/offsets` request through the Kafka Connect Rest API.
```

Comment:
I would leave this out. We want to promote the Strimzi way and not reference the REST API and get questions how to use it.